src/arch/helperrvv.h

```diff
@@ -292,8 +341,11 @@ static INLINE vfloat figetd_vf_di(fi_t d) {
 static INLINE vint2 figeti_vi2_di(fi_t d) {
   return SLEEF_RVV_SP_VGET_VI2(d, 1);
 }
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wuninitialized"
 static INLINE fi_t fisetdi_fi_vf_vi2(vfloat d, vint2 i) {
   fi_t res;
+  //res = SLEEF_RVV_SP_VLMUL_EXT_VI2(SLEEF_RVV_SP_VREINTERPRET_VI2(d));
```

Comment:
Remove leftovers. Same for the `#pragma` right above and the other comments below.

Reply:
This is where I started to fix the compilation warnings, so rather than remove them I doubled down and finished what I had started.  Yay no compiler warnings!